Private/GetNewCluster.ps1

```diff
@@ -35,7 +35,7 @@ Function GetNewClusterCluster {
     If (($PSBoundParameters.ContainsKey('NodeType')) -and ($NodeType)) { $Body['node_type_id'] = $NodeType }
     If (($PSBoundParameters.ContainsKey('DriverNodeType')) -and ($DriverNodeType)) { $Body['driver_node_type_id'] = $DriverNodeType }
     
-    If ($MinNumberOfWorkers -gt 0){
```

Comment:
The test that uses piped json files is failing with this change. I think this is what this was originally for. Can you change this line to be:
```
If (($MinNumberOfWorkers -ge 0) -and (-not $ClusterObject)){
```
So that it does not fire if values are piped and think this is good to go.

Reply:
Thanks for looking into this! :)
Applied in last commit